### ISSUE TITLE
Document the 5 main features of the add-on

### ISSUE THREAD

**Issue by NathanLovato** [GDquest/Blender-power-sequencer#13]
For v1.0, not every function needs docs. Let's focus on the most useful ones, that'll be part of the initial menu, and add other operators as they get improvements. E.g. the tools that work with markers aren't as useful without extra features, like generating titles from blender and importing external blend files are in.

- [x] Import local footage
- [ ] Render videos for the web
- [ ] Add and edit crossfade
- [ ] Cut and trim with the mouse
- [ ] Add and clear fade

**Comment by NathanLovato** [comment]
Some feedback from the community. After sending snippets for review, for each feature/section, we should have:

- A **brief description** of the tool/area and what problem it solves. Ideally hint it in the title/sum it up in a one-liner
- a paragraph to explain how it works and what it does
- a gif to show how it works. Use [a program to resize ](www.brianapps.net/sizer4) the blender window to 480p (854*480) so that all pictures have the same size. Don't capture the header. ShareX can do that precisely
- Techniques, a series of step by step explanations on how to use the tool by itself or with others to solve specific problems. 

This to-the-point and structured approach to the docs should help turn it into a learning guide more so than a purely descriptive document.

Example:

```md
# Toggle Mute Sequences to Edit Faster

Alt Click a sequence to toggle it muted with the mouse. This leaves sequences in the channel so when you trim audio or footage, it will snap to the muted strip. It's very useful when you're editing audio separately from the video, which is common with tutorials and other screencasts. It also works during playback and leaves the time cursor running.

## Techniques

### Cut and mute hickups on the fly

It's common to have pops, or little coughs in the audio. Often they're so short, all you'll want to do is to mute them.

1. Ctrl click before and after the incriminated sound on the waveform to cut a new strip.
2. Alt click on the new short sequence to mute it.
```

**Comment by davcri** [comment]
There is a little error in the link to `sizer4`, this is the correct URL: http://www.brianapps.net/sizer4

As it works only on Windows, here it is a way to do it on GNU\Linux with `wmctrl` ([Ubuntu package's page](https://packages.ubuntu.com/xenial/wmctrl)):
``` bash
# if Blender is maximized, the following command might not work
wmctrl -r Blender -e 0,0,0,640,480
```

**Comment by NathanLovato** [comment]
Just added initial pages for the wiki https://github.com/GDquest/Blender-power-sequencer/wiki
Anyone can now clone and contribute to it

**Comment by NathanLovato** [comment]
I started recording short video tutorials for the docs, gonna release them in batch on the Youtube channel. As promised the docs will go beyond the add-on and also cover Sequencer editing tricks. The videos are meant to be short and to the point.

@davcri are you still interested in giving a hand with the docs? The videos could use a quick review before going live. I've also written notes along the way, but can't handle code + videos + text-based docs. The project could really use a hand on this side.

![image](https://user-images.githubusercontent.com/12694995/33601553-e5858e28-d9ac-11e7-842f-971ca8353e68.png)

**Comment by davcri** [comment]
Yes interested, but I don't have much experience with Blender VSE: if this doesn't bother you, count me in 👍 I'll watch the videos, tinker with the extension and then start preparing the docs!

**Comment by NathanLovato** [comment]
Upload in progress, sending you the first vids in a few moments

**Comment by NathanLovato** [comment]
There you go! [Youtube playlist](https://www.youtube.com/playlist?list=PLhqJJNjsQ7KFjp88Cu57Zb9_wFt7nlkE) and [todo wiki page](https://github.com/GDquest/Blender-power-sequencer/wiki/todo)

There's a lot to do, and work on multiple fronts. Feel free to pick one feature, one task you'd like to tackle. Feel free to open them directly in the issue tracker. E.g. `[DOCS] Add crossfade`. If you need anything, I'm available to chat on Discord ( gdquest #4637). Most features won't take too long to document, and you'll see on the wiki page there are some paragraphs that just need some polish and maybe a picture or two.
I'm at your disposal if you have any questions.
